File: AvatarGameBoot/src/main/java/formation/sopra/AvatarGameBoot/restController/Param/Param.java
package formation.sopra.AvatarGameBoot.restController.Param;

import formation.sopra.AvatarGameBoot.entities.avatar.Avatar;
import formation.sopra.AvatarGameBoot.entities.item.Item;
import formation.sopra.AvatarGameBoot.entities.user.Users;

public class Param {
	
	private Users user;
	private Avatar avatar;
	private Item h;
	private Item b;
	private Item lh;
	private Item rh;
	private Item ll;
	private Item rl;
	
	
	
	
	public Users getUser() {
		return user;
	}
	public void setUser(Users user) {
		this.user = user;
	}
	public Param() {
		super();
	}
	public Avatar getAvatar() {
		return avatar;
	}
	public void setAvatar(Avatar avatar) {
		this.avatar = avatar;
	}

	public Item getH() {
		return this.h;
	}
	public void setH(Item h) {
		this.h = h;
	}
	public Item getB() {
		return this.b;
	}
	public void setB(Item b) {
		this.b = b;
	}
	public Item getLh() {
		return lh;
	}
	public void setLh(Item lh) {
		this.lh = lh;
	}
	public Item getRh() {
		return rh;
	}
	public void setRh(Item rh) {
		this.rh = rh;
	}
	public Item getLl() {
		return ll;
	}
	public void setLl(Item ll) {
		this.ll = ll;
	}
	public Item getRl() {
		return rl;
	}
	public void setRl(Item rl) {
		this.rl = rl;
	}


	
}

File: AngularAvatarGame/src/app/entities/param.ts
import { Avatars } from "./avatars";
import { Items } from "./items";
import { Users } from "./users";

export class Param {
    constructor(
        private _users?: Users,
        private _avatar?: Avatars,
        private _h?: Items,
        private _b?: Items,
        private _lh?: Items,
        private _rh?: Items,
        private _ll?: Items,
        private _rl?: Items,

    ) {}

    public get users(): Users | undefined {
        return this._users;
        }
    public set users(value: Users | undefined) {
        this._users = value;
    }

    public get avatar(): Avatars | undefined {
        return this._avatar;
        }
    public set avatar(value: Avatars | undefined) {
        this._avatar = value;
    }

    public get h(): Items | undefined {
        return this._h;
        }
    public set h(value: Items | undefined) {
        this._h = value;
    }

    public get b(): Items | undefined {
        return this._b;
        }
    public set b(value: Items | undefined) {
        this._b = value;
    }

    public get lh(): Items | undefined {
        return this._lh;
        }
    public set lh(value: Items | undefined) {
        this._lh = value;
    }

    public get rh(): Items | undefined {
        return this._rh;
        }
    public set rh(value: Items | undefined) {
        this._rh = value;
    }

    public get ll(): Items | undefined {
        return this._ll;
        }
    public set ll(value: Items | undefined) {
        this._ll = value;
    }

    public get rl(): Items | undefined {
        return this._rl;
        }
    public set rl(value: Items | undefined) {
        this._rl = value;
    }
}
